src/benchq/compilation/jabalizer_wrapper.jl

```diff
@@ -97,25 +57,31 @@ function prepare(num_qubits, qubit_map, icm_output)
     state
 end
 
-print("Input ICM circuit\n\t")
-@time (n_qubits, circuit) = icm_input("icm_input_circuit.json")
-
-print("ICM compilation: qubits=$n_qubits, gates=$(length(circuit))\n\t")
-@time (icm_output, data_qubits_map) = icm_compile(circuit, n_qubits)
+function run_jabalizer(circuit)
+    # Convert to Julia values, and (as a temp hack) rename T_Dagger to T (until implemented)
+    n_qubits = Jabalizer.pyconvert(Int, circuit.n_qubits)
+    icm_input = Jabalizer.ICMGate[]
+    for op in circuit.operations
+        name = Jabalizer.pyconvert(String, op.gate.name)
+        # TODO: THIS IS NOT CORRECT! TEMPORARY HACK!
+        name == "T_Dagger" && (name = "T")
```

Comment:
I think we could get rid of that, cause we should be able to handle `T_Dagger` in the new Jabalizer, right?

Reply:
T is actually handled in the ICM code (where it is changed to the CNOT)